test/e2e.sh

```diff
@@ -41,3 +47,4 @@ cleanup(){
 build
 up
 test
+dumplogs || true
```

Comment:
dumplogs should probably be in the cleanup trap, otherwise if test or up fails we won't dump logs?

Reply:
done